public/stylesheet/pumpio.css

```diff
@@ -12,6 +12,13 @@ body {
   color: white;
 }
 
+/* main */
+
+.hero-unit .span4 img {
+  max-height: 235px;
+  max-width: 340px;
```

Comment:
My CSS is a little shaky; `max-height` and `max-width` don't cause the image to be stretched or anything like `width`/`height` do, right? Just scaled?

Reply:
no, the max-(height/width) keep proportions and keep the size when is smaller than the maximum size (not force the size)